src/internal/metrics/metrics.go

```diff
@@ -376,11 +376,7 @@ func (r *Reporter) internalMetrics(metrics *Metrics) {
 		}
 		var sz, mbranch uint64 = 0, 0
 		for _, ri := range repoInfos {
-			if (sz + ri.SizeBytes) < sz {
-				sz = 0xFFFFFFFFFFFFFFFF
-			} else {
-				sz += ri.SizeBytes
-			}
+			sz += uint64(ri.SizeBytesUpperBound)
```

Comment:
Do we want to move this off of uint64 as well?

Reply:
Yeah, there are several other places where we are using uint64 that I would like switched over, but that would have to be in a followup or else this PR is going to get large. I focused on getting the PFS protos moved off of uint64 first.